pom.xml

```diff
@@ -35,11 +35,12 @@
   </scm>
 
   <properties>
-    <revision>6.20</revision>
+    <revision>7.0</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jetty.version>10.0.20</jetty.version>
+    <jetty.version>12.0.9</jetty.version>
     <slf4j.version>2.0.13</slf4j.version>
     <gitHubRepo>jenkinsci/winstone</gitHubRepo>
+    <maven.compiler.release>17</maven.compiler.release>
```

Comment:
This is working around the fact that the parent POM doesn't (yet!) require Java 17 [JENKINS-73121](https://issues.jenkins.io/browse/JENKINS-73121). Since we might forget to remove this workaround once JENKINS-73121 is resolved, let's leave a TODO comment here referencing JENKINS-73121.

```suggestion
    <!-- TODO JENKINS-73121 until in parent POM -->
    <maven.compiler.release>17</maven.compiler.release>
```